lib/langchain/vectorsearch/weaviate.rb

```diff
@@ -71,6 +71,15 @@ def update_texts(texts:, ids:)
       end
     end
 
+    def remove_texts(ids:)
```

Comment:
Thoughts on using the [batch_delete](https://github.com/andreibondarev/weaviate-ruby/blob/main/lib/weaviate/objects.rb#L160) method?

Reply:
Sorry didn't see this, wanted to check on what would be the right path for the `where `param, I had something like this
`{ path: [\"__id\"], operator: Equal, valueString: \"#{ids}\" `?

Reply:
Batch delete docs: https://weaviate.io/developers/weaviate/api/rest/batch#batch-delete

But take a look at the where filter specs: https://weaviate.io/developers/weaviate/api/graphql/filters#where-filter

You may need to use `"ContainsAny":
![image](https://github.com/andreibondarev/langchainrb/assets/541665/7b17366c-6e90-4394-88f7-57c46f3c6b7e)